lib/presentation/di/module/widget_module.dart

```diff
@@ -152,7 +152,8 @@ class WidgetModule {
     getIt.registerFactory(() => SharedSpaceWidget());
     getIt.registerFactory(() => SharedSpaceViewModel(
       getIt.get<Store<AppState>>(),
-      getIt.get<GetAllSharedSpacesInteractor>()
+      getIt.get<GetAllSharedSpacesInteractor>(),
+      getIt<SearchSharedSpaceNodeNestedInteractor>()
```

Comment:
What is the difference between these two ways ? :)

Reply:
there is no difference at all, you should use `getIt<...>()` because it shorter :)